ng-dev/pr/config/index.ts

```diff
@@ -76,7 +79,8 @@ export function assertValidPullRequestConfig<T extends NgDevConfig>(
     );
   }
 
-  if (config.pullRequest.githubApiMerge === undefined) {
+  const {githubApiMerge} = config.pullRequest;
+  if (githubApiMerge === undefined) {
```

Comment:
```suggestion
  if (config.pullRequest.githubApiMerge === undefined) {
```

I think we can just go back to the check we had before right?